.github/workflows/bare-diffs.yml

```diff
@@ -0,0 +1,53 @@
+name: Check Tempalte Bare Minimum Diffs
+
+on:
+  workflow_dispatch: {}
+  push:
+    branches: [main]
+    paths:
+      - 'templates/expo-template-bare-minimum/**'
+      - '.github/workflows/bare-diffs.yml'
+  pull_request:
+    paths:
+      - 'templates/expo-template-bare-minimum/**'
+      - '.github/workflows/bare-diffs.yml'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  bare-diffs:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: 👀 Checkout
+        uses: actions/checkout@v3
+      - name: ⬢ Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+      - name: ♻️ Restore caches
+        uses: ./.github/actions/expo-caches
+        id: expo-caches
+        with:
+          yarn-docs: 'true'
+      - name: ➕ Add `bin` to GITHUB_PATH
+        run: echo "$(pwd)/bin" >> $GITHUB_PATH
+      - name: 🧶 Yarn install
+        if: steps.expo-caches.outputs.yarn-docs-hit != 'true'
+        run: yarn install --frozen-lockfile
```

Comment:
not sure the line here is running `yarn install` in workspace root? is that necessary?
if the answer is yes, we could add `yarn-workspace: 'true'` in expo-caches to speed up the yarn installation.